src/main/java/de/hysky/skyblocker/config/categories/DungeonsCategory.java

```diff
@@ -320,6 +315,14 @@ public static ConfigCategory create(SkyblockerConfig defaults, SkyblockerConfig
 								newValue -> config.locations.dungeons.enableMap = newValue)
 						.controller(ConfigUtils::createBooleanController)
 						.build())
+				.option(Option.<Boolean>createBuilder()
+						.name(Text.translatable("text.autoconfig.skyblocker.option.locations.dungeons.dungeonScore.enableScoreHUD"))
+						.description(OptionDescription.of(Text.translatable("text.autoconfig.skyblocker.option.locations.dungeons.dungeonScore.enableScoreHUD.@Tooltip"), Text.translatable("text.autoconfig.skyblocker.option.locations.dungeons.dungeonScore.enableScoreHUD.deathMessagesNote")))
+						.binding(defaults.locations.dungeons.dungeonScore.enableScoreHUD,
+								() -> config.locations.dungeons.dungeonScore.enableScoreHUD,
+								newValue -> config.locations.dungeons.dungeonScore.enableScoreHUD = newValue)
+						.controller(ConfigUtils::createBooleanController)
+						.build())
```

Comment:
I think these should go in the dungeon score category. If you disagree you can put it in both.